demo-app/src/utils/locations.js

```diff
@@ -1,4 +1,4 @@
-const statuses = ['good', 'warning', 'critical'];
+const statuses = ['good', 'warning', 'critical', undefined];
```

Comment:
Having undefined explicitly as an array item feels a little strange. I wonder if we can adjust the logic in the markers to avoid needing this?

Reply:
Fixed in latest commit